hoodie-common/src/main/java/com/uber/hoodie/common/model/FileSlice.java

```diff
@@ -50,8 +50,15 @@
    */
   private final TreeSet<HoodieLogFile> logFiles;
 
-  public FileSlice(String baseInstantTime, String fileId) {
-    this.fileId = fileId;
+  public FileSlice(String partitionPath, String baseInstantTime, String fileId) {
+    this.fileGroupId = new HoodieFileGroupId(partitionPath, fileId);
```

Comment:
you can call `this(new HoodieFileGroupId(..),...)` and save some lines of code?

Reply:
Fixed.